osu.Game/Screens/Play/HUD/ClicksPerSecond/ClicksPerSecondCounter.cs

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Localisation;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Skinning;
+using osuTK;
+
+namespace osu.Game.Screens.Play.HUD.ClicksPerSecond
+{
+    public class ClicksPerSecondCounter : RollingCounter<int>, ISkinnableDrawable
+    {
+        private const float alpha_when_invalid = 0.3f;
+
+        private readonly Bindable<bool> valid = new Bindable<bool>();
```

Comment:
I'm not sure the purpose of this but it shouldn't exist.

Reply:
I guess I was little bit too inspired by `UnstableRateCounter` 😅

https://github.com/ItsShamed/osu/blob/kps/osu.Game/Screens/Play/HUD/ClicksPerSecond/ClicksPerSecondCounter.cs#L47-L49

But I guess it still stays at 0 even if there is no calculator (which is unlikely to happen anyways), so I'll remove it.